dash-pipeline/Makefile

```diff
@@ -62,8 +62,6 @@ SAI/SAI: sai-submodule
 sai-submodule:
 	@echo "Initializing SAI submodule..."
 	git submodule update --init
-	@echo "Initializing SAI submodule..."
```

Comment:
Please revert. See main comments for reason.

Reply:
removed